packages/same-structure/src/sameStructure.js

```diff
@@ -165,12 +165,13 @@ function pathStr(path) {
 // mustBeSameStructureInternal
 function mustBeSameStructureInternal(left, right, message, path) {
   function complain(problem) {
-    const template = harden([
-      `${message}: ${problem} at ${pathStr(path)}: (`,
-      ') vs (',
-      ')',
-    ]);
-    insist(false)(template, left, right);
+    assert.fail(
+      details(
```

Comment:
Still too much magic for me. At least please put in a comment explaining the template magic. I couldn't maintain this code when I was changing the formats.

Reply:
I agree. I'll comment this in assert and put comment citations everywhere I do this.

But running out of steam tonight. Will resume tomorrow.

Reply:
Fixed via `openDetail`. Done.